.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: python
 python:
   - "2.7"
+  - "3.4"
+matrix:
+  allow_failures:
+    - python: "3.4"
```

Comment:
I know you are using `3.4` because `3.6` is failing. But since `3.4` is failing as well, should we just use `3.6`?